site/load-chart.js

```diff
@@ -191,6 +192,23 @@ function setupChart () {
 
             area_dropdown.onchange = update_chart
             period_dropdown.onchange = update_chart
+            if (window.matchMedia) {
+                window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change',({ matches }) => {
```

Comment:
```suggestion
                window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change', ({ matches }) => {
```

`npm run lint` complains about this.